.github/settings.yml

```diff
@@ -0,0 +1,12 @@
+repository:
+  name: solana-solidity.js
+  default_branch: master
+  has_downloads: false
+  has_issues: true
```

Comment:
Will we still allow new issues in this repository?

Reply:
This is what an archived repo looks like https://github.com/hyperledger/burrow